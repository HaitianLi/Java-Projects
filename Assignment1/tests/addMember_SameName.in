add member jack hu
add member jack hu
member 100000
member 100001
member 100002
exit