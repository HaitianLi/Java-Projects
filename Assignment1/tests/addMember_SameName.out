user: Success.

user: Success.

user: 100000: jack hu

user: 100001: jack hu

user: No such member in system.

user: Ending Library process.
